pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "prometheus-eaton-ups-exporter"
+authors = [
+    {name = "Mathis Lövenich"},
+]
+description = "A Prometheus exporter for Eaton UPSs."
+version = "1.2.0"
+readme = "README.md"
+requires-python = ">=3.11"
+keywords = ["prometheus", "exporter", "eaton"]
+license = {text = "ISC"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
```

Comment:
`requirements.txt` and `test-requirements.txt` should be folded into this file and removed.